Components/Grid/grid.css

```diff
@@ -0,0 +1,35 @@
+.grid {
+  display: grid;
+  gap: 1rem;
+  width: 50rem;
+  height: 5rem;
+}
+
+.grid_w_two-items {
+  grid-template-areas: "ItemOne ItemTwo";
```

Comment:
Hey,
Your implementation and logic looks on point, but the idea of Grid system is to provide a Row and Column sort of matrix.
Your examples here are only showing a single row with multiple columns which can already be easily achieved by using Flex alone.
I'd suggest you to modify grid-template-areas to accomodate multiple rows as well